tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -229,6 +229,11 @@
               {{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
               {%- endfor %}
               {% set min_viable_mtu = mtu_list | max %}
+              {% set os_net_config_routes = dataplane_os_net_config_routes | default(ctlplane_host_routes) %}
+              {% set os_net_config_iface = dataplane_os_net_config_iface | default('nic1') %}
+              {% set neutron_public_iface = dataplane_public_iface | default('eth1') %}
+
```

Comment:
this template consumes the ansible vars from the dataplane inventory generated by the dataplane-operator, so for having these variables injected there, you need to add them here under `ansibleVars`